.github/workflows/test.yml

```diff
@@ -35,10 +35,12 @@ jobs:
         run: "helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx"
 
       - name: Create KIND Cluster
-        uses: helm/kind-action@v1.2.0
+        uses: helm/kind-action@94729529f85113b88f4f819c17ce61382e6d8478 # pin@v1.2.0
 
       - name: Install Ingress Controller
-        run: "helm install ingress-nginx/ingress-nginx --generate-name --set controller.service.type='NodePort' --set controller.admissionWebhooks.enabled=false"
+        run: "helm install ingress-nginx/ingress-nginx --generate-name --set
```

Comment:
@developer-guy if this is going to be broker out into multiple lines, can the `--set` align?

Reply:
ah yes, sorry about that, it is my formatter bad 🤦‍♂️